### ISSUE TITLE
J6 uses of schema:isRelatedTo

### ISSUE THREAD

**Issue by xgmachina** [USGCRP/gcis-ontology#108]
When relating a file to a publication, is schema:isRelatedTo the correct property to use?  See e.g.: https://data.globalchange.gov/file/c18de699-ad4e-4055-8557-58ac989fe18c.thtml

**Comment by xgmachina** [comment]
The domain and range of `schema:isRelatedTo` are both `schema:Product`. If we regard the PDF and the publication both as product then it should be okay to use `schema:isRelatedTo`  there.

**Comment by congruili** [comment]
do we want to make pdf and publication some subclasses of `schema:Product`, or we just want to treate them as instances of `schema:Product`?

**Comment by justgo129** [comment]
@zednis what do you recommend?

**Comment by zednis** [comment]
I think I need some background.  What relationship does the file have to the report?  Perhaps an example would help.

**Comment by justgo129** [comment]
The "file" represents the downloadable PDF (or extension) which represents the report.  e.g.:
https://data.globalchange.gov/report/aeo2013

click on the thumbnail to get:
https://data.globalchange.gov/file/d25e1cfd-e12e-4101-8402-5d09e68d7a1c

Clicking on this downloadable thumbnail produces the report. Hence the file has a SHA-1, MIME type, etc.  Please also note the sentence at the bottom of the page accessible by the latter URL: "This file is associated with..."  That is the sentence of interest for the purpose of this issue.

**Comment by justgo129** [comment]
and "prov:wasAssociatedWith" is specific to activities, not to characterizing an association between two entities.  Hope this clarifies matters.

**Comment by zednis** [comment]
So the file is a representation of the report.  The file is not just associated with the report, the file is a representation of the report.

**Comment by justgo129** [comment]
That makes sense.  @rewolfe?

**Comment by zednis** [comment]
This reminds me of the FRBR model.  

https://en.wikipedia.org/wiki/Functional_Requirements_for_Bibliographic_Records

What you are calling the instance of type 'File' is (in FRBR) a manifestation of the report.

**Comment by justgo129** [comment]
So something like frbr:embodimentOf?  http://vocab.org/frbr/core.html

http://data.globalchange.gov/file/9bdd68ab-4b90-4ad9-b043-fedb7e564184
   dcterms:identifier "9bdd68ab-4b90-4ad9-b043-fedb7e564184";
   gcis:cryptographicHash [ 
      gcis:hashFunction dbpedia:SHA-1;
      rdf:value "bdbe9f11d4b77589c541c062cfdeca636440dd82" ] ;
   dcterms:format "application/pdf";
   frbr:embodimentOf http://data.globalchange.gov/report/2014car;

**Comment by justgo129** [comment]
where the URI for the "file" appears in the open brackets?

**Comment by justgo129** [comment]
with URIs surrounded by <<>> accordingly

**Comment by zednis** [comment]
If we wanted to use the FRBR property directly I believe that is the correct encoding in RDF.

```
<http://data.globalchange.gov/file/9bdd68ab-4b90-4ad9-b043-fedb7e564184>
  dcterms:identifier "9bdd68ab-4b90-4ad9-b043-fedb7e564184";
  gcis:cryptographicHash [ 
    gcis:hashFunction dbpedia:SHA-1;
    rdf:value "bdbe9f11d4b77589c541c062cfdeca636440dd82" ] ;
  dcterms:format "application/pdf";
  frbr:embodimentOf <http://data.globalchange.gov/report/2014car> .
```

We could mint a sub-property of `frbr:embodimentOf` if the definition or terminology employed by FRBR causes confusion, but I believe our scenario translates to this mapping in FRBR.

I find FRBR to be a nice framework to discuss issues like this (the PDF vs the PDF content)

**Comment by justgo129** [comment]
+1

**Comment by justgo129** [comment]
Closed #108 due to merged https://github.com/USGCRP/gcis/pull/221.
